docs/source/fitting/fitfunctions/ChudleyElliotSQE.rst

```diff
@@ -0,0 +1,48 @@
+.. _func-Chudley-ElliotSQE:
+
+================
+ChudleyElliotSQE
+================
+
+.. index:: ChudleyElliotSQE
+
+Description
+-----------
+
+This fitting function models the dynamic structure factor
+for a particle undergoing jump diffusion using the Chudley-Elliot Jump diffusion model [1]_.
+
+.. math::
+
+   S(Q,E) = Height \cdot \frac{1}{\pi} \frac{\Gamma}{\Gamma^2+(E-Centre)^2}
+
+   \Gamma = \frac{\hbar}{\tau} \cdot \left(1 - \frac{sin(Ql)}{Ql}\right)
+
+where:
+
+-  :math:`Height` - Intensity scaling, a fit parameter
+-  :math:`Centre` - Centre of peak, a fit parameter
+-  :math:`Tau` - Residence time, a fit parameter
+-  :math:`l` - Jump lengths, a fit parameter
+-  :math:`Q` - Momentum transfer, an attribute (non-fitting)
+
+Units of :math:`l` are inverse units of :math:`Q`.
+
+Units of :math:`HWHM` are :math:`meV` if units of :math:`\tau` are *ps*.
+Alternatively, units of :math:`HWHM` are :math:`\mu eV` if units of
+:math:`\tau` are *ns*.
+
```

Comment:
It would be helpful if you included a figure that illustrates the ChudleyElliot fit function (also applies to Hall-Ross and FickDiffusion) for some typical values of fit parameters. This would help a reader who is unfamiliar with this function to get a better understanding on how it behaves without trying to find it in literature.

Reply:
The shape of the function will depend on the sample & how well it is fitted. The user may well try all 3 functions to determine which is best scietifically. Showing shapes serves no use.

Reply:
Alright